packages/sui-ssr/server/ssr/index.js

```diff
@@ -36,8 +36,16 @@ const useLegacyContext =
     ? ssrConfig.useLegacyContext
     : true
 
+const setContentType = res => {
+  if (ssrConfig.serverContentType !== 'undefined') {
+    res.set('Content-Type', ssrConfig.serverContentType)
+  } else {
+    res.type('html')
```

Comment:
A better approach would be to add a defaults values to config in order to avoid this kind of branches in the middle of the code.

Reply:
well as you can see one branch is using res.type and the other res.set and in order to keep backward compatibility, we've decided to go with this approach

Reply:
Indeed. What I see this branch could be easily avoided by using a default value to the `ssrConfig.serverContentType`. Why? If not, we're going to polute the whole code with branching everytime a value is missing.

We could add on the `config.js` the default value.

``` //config.js
const DEFAULT_VALUES = {
  serverContentType: 'html'
}

export default { ...DEFAULT_VALUES, ...ssrConfig }
```

Later, instead the if you could simply:
```
res.type(ssrConfig.serverContentType)
```

As mentioned on the documentation, type will set the Content-Type header when the string passed adds a `/`: https://expressjs.com/es/api.html#res.type

Reply:
Said that, I think is fair to merge this, as it's doing what's mean to do, and we could add later ;)

Reply:
I do not see how we can set the charset encoding using the approach you've mentioned. To add more context to this PR our problem is that we are using a charset encoding that is not `utf-8` and seems that using only `res.type('html')` by defaults sets the charset encoding to `utf-8` that's why we decided to use the res.set where you can explicitly set the charset encoding.

Reply:
Easy: `res.type('text/html;charset=windows-1252')`

Reply:
oh nice, I didn't understood what the documentation means to say with `If type contains the “/” character, then it sets the Content-Type to type.`